master/internal/api_auth.go

```diff
@@ -137,11 +137,19 @@ func processProxyAuthentication(c echo.Context) (done bool, err error) {
 		return true, err
 	}
 
+	snapshot := command.CommandSnapshot{}
```

Comment:
wdy think about modifying identifyTask to get the exp and trial lists? https://github.com/determined-ai/determined/pull/5828#discussion_r1088481970 instead of this new db call?
since/to:
- we ran into an issue that seemed to be about restoring snapshots on a tls enabled cluster 
- avoid adding another db query